src/components/Comments.jsx

```diff
@@ -0,0 +1,46 @@
+import{ useState, useEffect } from "react";
+import { getArticleComments } from "../api";
+
+const Comments = ({ article_id }) => {
+    const [comments, setComments] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+        setIsLoading(true);
+
+        getArticleComments(article_id)
+        .then((commentsData) => {
+            setComments(commentsData);
+            setIsLoading(false);
+        });
+    }, [article_id]);
```

Comment:
As this component looks to be rendered when an article is shown, the only way for the article to change would be for the url to change, if you have a useEffect for a single article (you should) then the page would re render along with this component. Therefore you need the array...but it doesn't necessarily need anything in it.